### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by ghost**
This pull request add the basic setup of the project:
  - Docker
  - Docker-compose
  - Make
  - travis.yml (Please, check if something is missing for travis config)
  - Basic Django auto-gen web project

* The current db uses is ~postgreSQL~ MariaDB

### CODE REVIEW

**Review by the-spelletier — COMMENTED** [review]
Tried to use the make commands with success! Basic project looks great.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM python:3.5
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update
```
Comment (gperiard) [inline Dockerfile]: You dont need to run this since you dont install any package

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+ - Docker*
+ - Docker-Compose
+ - Make
+
+*Configuring non-root access is recommanded
+
+### Starting Development Environement
+
+ - run `$ make` in root project directory.
+ - Access the app via your browser at 0.0.0.0:8000
```
Comment (gperiard) [inline README.md]: The app IP is not 0.0.0.0, it's localhost. 0.0.0.0 when you launch the app means that it listen on all interfaceas

File: djangoAssets/settings.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    },
+]
+
+WSGI_APPLICATION = 'djangoAssets.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/1.11/ref/settings/#databases
+
+DATABASES = {
```
Comment (gperiard) [inline djangoAssets/settings.py:76]: The database must be mysql/mariadb

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
```
Comment (gperiard) [inline docker-compose.yml]: Mysql/mariadb

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+django==1.10.2
+docker-compose==1.7.0
+psycopg2
```
Comment (gperiard) [inline requirements.txt]: You wont need this

**Review by gperiard — CHANGES_REQUESTED** [review]
Nice work, but there's a few details to change.

**Comment by ghost** [comment]
All change completed

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,25 @@
+all:
```
Comment (gperiard) [inline Makefile]: I think `run-dev` would be a better name

File: djangoAssets/settings.py [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '=#0irkgka4+eh_d9l6hkn&fi$v&i7^$_bof&3&$fi%)dedhq$_'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ['0.0.0.0']
```
Comment (aviau) [inline djangoAssets/settings.py]: - [ ] ``ALLOWED_HOSTS = ['0.0.0.0', 'localhost']``

**Review by gperiard — APPROVED** [review]

**Pull request merged**